src/main/java/picard/sam/AbstractAlignmentMerger.java

```diff
@@ -964,8 +1035,9 @@ public boolean isClipOverlappingReads() {
         return clipOverlappingReads;
     }
 
-    public void setClipOverlappingReads(final boolean clipOverlappingReads) {
+    public void setClipOverlappingReads(final boolean clipOverlappingReads, final boolean hardClipOverlappingReads) {
```

Comment:
as a change in public API, this is annoying. perhaps you should leave the previous method with a default of false?